### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by plankton9991**
[DEMO LINK](https://plankton9991.github.io/react_sum/)

### CODE REVIEW

File: src/components/Sum/Sum.spec.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { mount } from '@cypress/react';
-import { Sum } from './Sum';
+import Sum from './Sum';
```
Comment (YegorKochetkov) [inline src/components/Sum/Sum.spec.jsx:3]:
```suggestion
import { Sum } from './Sum';
```

File: src/components/Sum/Sum.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  a: 0,
+  b: 0,
+};
+
+Sum.propTypes = {
+  a: PropTypes.number,
+  b: PropTypes.number,
+};
+
+export default Sum;
```
Comment (YegorKochetkov) [inline src/components/Sum/Sum.jsx:21]:
```suggestion
```

File: src/components/Sum/Sum.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({
```
Comment (YegorKochetkov) [inline src/components/Sum/Sum.jsx:4]:
```suggestion
export const Sum = ({
```

**Review by YegorKochetkov — APPROVED** [review]
Ok, but don't change Sum.spec.jsx. 
enough add export to Sum constant instead default export

**Review by AlinaMatuschak — APPROVED** [review]

**Pull request closed without merging**